### ISSUE TITLE
Autocomplete does not work correctly when the src url contains an existing query parameter

### ISSUE THREAD

**Issue by JoyceBabu** [ampproject/amphtml#26522]
## What's the issue?

The dynamic loading option of the autocomplete plugin is not working, when the `src` url already containers a query string. For example, If I set the `src` to 'https://amp.dev/search.json' and the `query` attribute to `q`, then the requested url is 
```
https://amp.dev/search.json?q=<my search input>&__amp_source_origin=<origin>
```

But if I change the `src` to `https://amp.dev/search.json?param=1`, then the queried url is
```
https://amp.dev/search.json?param=1&__amp_source_origin=https%3A%2F%2Fnull.jsbin.com
```

## How do we reproduce the issue?

https://jsbin.com/kemihapuce/edit?html,output

1. Open Network tab in 'Developer Console'
2. Type in the input box
3. Check the url of the ajax requests triggered by autocomplete. The query is not present in the requested url.

## What browsers are affected?

All browsers

## Which AMP version is affected?
Latest

**Comment by caroqliu** [comment]
Hi, thanks for filing this issue! This is definitely not desired behavior--I'll work on a fix shortly.

**Comment by JoyceBabu** [comment]
@caroqliu If it is okay, shall I send a PR?

**Comment by caroqliu** [comment]
@JoyceBabu Yes, absolutely! Please feel free to add me as a reviewer. Thanks for contributing. :)
